pyproject.toml

```diff
@@ -38,9 +46,8 @@ starlette = "*"
 python-multipart = "*"
 rich = "*"
 pyyaml = "*"
-attrs = ">=20"
 cattrs = "*"
-importlib_metadata = "*"
+attrs = ">=21"
```

Comment:
This should be `>=21.1` for typing to work, and I think leaving things as they were is fine?

Reply:
ok lets open a follow up PR about this